draft-ietf-privacypass-protocol.md

```diff
@@ -422,6 +431,15 @@ opaque PublicKey<1..2^16-1>
 opaque PrivateKey<1..2^16-1>
 ~~~
 
+### Public Metadata {#pp-public-metadata}
+
+We use the following type to describe the metadata that can be added
+by the server and/or client.
+
+~~~
+opaque Metadata<1..2^16-1>
```

Comment:
This means that metadata is length-prefixed, which isn't the case as described in text above. Maybe we just say that it's an arbitrary-length byte string and leave it at that? Then we could delete this section entirely. (See the proposed suggestions below!)

Reply:
Yeah.. I didn't know how to make it as long as wanted in the TLS language. Changed now ;)